### DISCUSSION TITLE
[Released] Expose timezone information in functions

### DISCUSSION THREAD

**Discussion by cj** [vercel/community#94] [Idea]
It would be great if we could get access to the timezone of the incoming request.

**Comment by Pieparker** [comment]
Thanks for the feedback. I've +1'd this discussion to an internal feature request tracker. If the team decide to pursue this and there's any progress, we'll be sure to loop back!

**Reply by cj** [reply]
Thank you!

**Comment by Pieparker** [comment]
Hey @cj, the team have shipped this feature request!

We have added timezone to both serverless functions and edge functions/middleware. The value is like `Asia/Tokyo` instead of `+hhmm`. The header name is `x-vercel-ip-timezone`.

You can read more in the docs [here](https://vercel.com/docs/concepts/edge-network/headers#x-vercel-ip-timezone). As ever, thank you for sharing this request and making this a reality!
